### ISSUE TITLE
Downloading IXI dataset: link broken

### ISSUE THREAD

**Issue by jeandut** [owkin/FLamby#253]
When running the download script in the IXI folder I now get:
`python download.py --output-folder ./IXI`
````
The IXI dataset is made available under the Creative Commons CC BY-SA 3.0 license.
    If you use the IXI data please acknowledge the source of the IXI data, e.g. the following website: https://brain-development.org/ixi-dataset/
    IXI Tiny is derived from the same source. Acknowledge the following reference on TorchIO : https://torchio.readthedocs.io/datasets.html#ixitiny
    Pérez-García F, Sparks R, Ourselin S. TorchIO: a Python library for efficient loading, preprocessing, augmentation and patch-based sampling of medical images in deep learning. arXiv:2003.04696 [cs, eess, stat]. 2020. https://doi.org/10.48550/arXiv.2003.04696
Traceback (most recent call last):
  File "download.py", line 81, in <module>
    dl_ixi_tiny(args.output_folder, args.debug)
  File "download.py", line 47, in dl_ixi_tiny
    response.raise_for_status()
  File "/Users/jeanduterrail/anaconda3/envs/flamby_test/lib/python3.8/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://md-datasets-cache-zipfiles-prod.s3.eu-west-1.amazonaws.com/7kd5wj7v7p-1.zip
````

**Comment by jeandut** [comment]
@AyedSamy and @sssilvar could you take a look when you have time ?

**Comment by jeandut** [comment]
Seeing this file it looks ike IXI Tiny can be found there: https://www.dropbox.com/s/ogxjwjxdv5mieah/ixi_tiny.zip?dl=1

**Comment by jeandut** [comment]
However simply changing the URL creates this new error when using benchmarking:
Traceback (most recent call last):
```
  File "benchmark.py", line 169, in <module>
    main(
  File "benchmark.py", line 49, in main
    FedIXITiny(transform=training_transform, train=True, pooled=True),
  File "/Users/jterrail/Downloads/FLamby/flamby/datasets/fed_ixi/dataset.py", line 206, in __init__
    super(FedIXITiny, self).__init__(
  File "/Users/jterrail/Downloads/FLamby/flamby/datasets/fed_ixi/dataset.py", line 85, in __init__
    for subject in os.listdir(self.subjects_dir)
FileNotFoundError: [Errno 2] No such file or directory: '/Users/jterrail/Desktop/FLamby/flamby/datasets/fed_ixi/dataset_creation_scripts/IXI/7kd5wj7v7p-1/IXI_sample'
```

**Comment by jeandut** [comment]
Should be solved now in #257 again thank you both for the help.
